templates/incl/editor-and-output.html

```diff
@@ -1,180 +1,230 @@
 {% set editor_height='22rem' %}
 <script src="https://cdn.jsdelivr.net/npm/js-confetti@latest/dist/js-confetti.browser.js"></script>
-    <div id="editor-area" class="relative grid grid-cols-1 md:grid-cols-2 md:grid-flow-row-dense gap-4">
+<div id="editor-area" class="relative grid grid-cols-1 md:grid-cols-2 md:grid-flow-row-dense gap-4">
 
-    <div id="fold_out_toggle_container" class="hidden absolute z-10 top-16 ltr:-left-2 rtl:-right-2 cursor-pointer" onclick="hedyApp.show_editor();">
-     <div class="bg-green-500 text-white py-3 rounded-lg text-sm">
-         <span>→</span>
-     </div>
+    <div id="fold_out_toggle_container" class="hidden absolute z-10 top-16 ltr:-left-2 rtl:-right-2 cursor-pointer"
+         onclick="hedyApp.show_editor();">
+        <div class="bg-green-500 text-white py-3 rounded-lg text-sm">
+            <span>→</span>
+        </div>
     </div>
     <!-- Row 1: editor, output pane -->
-    <div data-devmodeheight="{{editor_height}},36rem" class="flex flex-col order-1 relative" id="code_editor" style="height: {{editor_height}}">
-        <div id="fold_in_toggle_container" class="hidden md:flex absolute z-10 top-16 ltr:-right-1 rtl:-left-1 cursor-pointer" onclick="hedyApp.hide_editor();">
-         <div id="text_field" class="bg-green-500 text-white py-3 rounded-lg text-sm">
-             <span>←</span>
-         </div>
+    <div data-devmodeheight="{{ editor_height }},36rem" class="flex flex-col order-1 relative" id="code_editor"
+         style="height: {{ editor_height }}">
+        <div id="fold_in_toggle_container"
+             class="hidden md:flex absolute z-10 top-16 ltr:-right-1 rtl:-left-1 cursor-pointer"
+             onclick="hedyApp.hide_editor();">
+            <div id="text_field" class="bg-green-500 text-white py-3 rounded-lg text-sm">
+                <span>←</span>
+            </div>
         </div>
         <!--The div below is used to send translation of search to Ace through the frontend-->
-        <div id="search_placeholder" style="display: none":>{{_('search')}}</div>
+        <div id="search_placeholder" style="display: none" :>{{ _('search') }}</div>
         <!-- Status warning -->
         <div data-view="if-submitted"
-          class="hidden flex-0 z-20 mt-0 bg-yellow-100 border-b-4 border-yellow-500 rounded-b text-yellow-900 px-4 py-3 shadow-md">
-          {{_('submitted_header')}}
+             class="hidden flex-0 z-20 mt-0 bg-yellow-100 border-b-4 border-yellow-500 rounded-b text-yellow-900 px-4 py-3 shadow-md">
+            {{ _('submitted_header') }}
```

Comment:
I think you used a tool to autoformat here? Maybe your IDE does that accidentally? But this (as Hasan explained) makes it harder to review the PR, so if you can revert it, please do!